ansible/deploy_mailrelay.yml

```diff
@@ -0,0 +1,27 @@
+- name: "Install postfix"
+  hosts: all
+  sudo: yes
+  tasks:
+    - package: name=postfix state=present
+
+- name: "mail relay deploy"
+  hosts: mailrelay
+  sudo: yes
+  tasks:
+    - name: "sets our relayhost"
+      lineinfile:
+        dest: /etc/postfix/main.cf
+        regexp: '^relayhost ='
+        line: "relayhost = {{ mailrelay }}"
```

Comment:
Just to avoid the confusion between actual `mailrelay` host and the postfix (or 'proxy') `mailrelay` host variables, can we rename them on the lines of `mailrelay_postfix_host` and `mailrelay_host` or some other name?